spec/server/jenkins_master/jenkins_master_spec.rb

```diff
@@ -47,6 +47,11 @@
           its(:stdout) { should match '{}' }
         end
       end
+      # Same trick but this time with api/python
+      describe command("curl --verbose --insecure -A \"#{agent}\" -H 'Location: https://ci.jenkins.io/' https://127.0.0.1/api/python") do
+        its(:exit_status) { should eql 0 }
```

Comment:
Crazy. Should be a 403 I think.

Reply:
@rtyler can correct me if I'm wrong, but I believe this approach is because doing a 403 doesn't necessarily shut up some of these shitty clients - they just keep trying.

Reply:
correctomundo! :frowning:

Reply:
So can we not just continue sending 403s?

Reply:
Every response that's not 200 with fake data will just result in reattempted requests from terrible clients.